kolibri/plugins/coach/assets/src/views/exams/exam-report-page/index.vue

```diff
@@ -148,6 +145,12 @@
           reportGroupings = this.learnerGroups.map(group =>
             this.examTakers.filter(learner => learner.group.id === group.id)
           );
+          reportGroupings = orderBy(
+            reportGroupings,
+            [grouping => grouping[0].group.name.toUpperCase()],
```

Comment:
Appears to cause a problem if reportGrouping[0] is undefined, which can happen if you have a Learner Group with zero members.